Sources/PackageModel/SwiftSDKs/SwiftSDKConfigurationStore.swift

```diff
@@ -158,7 +158,15 @@ public final class SwiftSDKConfigurationStore {
         return true
     }
 
-    public func configure(
+    /// Configures the specified SDK and identified target triple with the configuration parameter.
+    /// - Parameters:
+    ///   - sdkID: ID of the Swift SDK to operate on.
+    ///   - tripleString: run-time triple for which the properties should be configured, or nil to configure all triples for the SDK
```

Comment:
"Swift SDK" is a SwiftPM concept, while "SDK" is an Xcode and partially a Clang concept, this is also reflected in the difference between `--sdk` and `--swift-sdk` CLI options. We shouldn't mix these two up and should always refer to Swift SDKs as "Swift SDK".

```suggestion
    /// Configures the specified Swift SDK and identified target triple with the configuration parameter.
    /// - Parameters:
    ///   - sdkID: ID of the Swift SDK to operate on.
    ///   - tripleString: run-time triple for which the properties should be configured, or nil to configure all triples for the Swift SDK
```